src/reducers/index.js

```diff
@@ -1,7 +1,7 @@
-import undoable, { excludeAction } from 'redux-undo'
+import undoable, { excludeAction, groupByActionTypes } from 'redux-undo'
 import iprove from './iprove'
-import { ADD_IDENTIFIERS, ADD_ATOMS, ADD_RELATIONS } from '../constants';
+import { ADD_IDENTIFIERS, ADD_ATOMS, ADD_RELATIONS, ADD_TYPES, UPDATE_RULE, SET_SCOPE, NEW_STEP } from '../constants';
```

Comment:
UPDATE_RULE,  SET_SCOPE, NEW_STEP are not being used

Reply:
Have removed them now